tox.ini

```diff
@@ -22,6 +22,13 @@ commands =
     coverage combine
     coverage report -m
 
+# test ssh extra on py36 only
+[testenv:py36]
```

Comment:
Let's actually just add this in the testenv extras (line 7) as `ssh: ssh`. Then you can add a single new job to .travis.yml that looks like:
```
        - python: 3.8 
          env: TOXENV=py38-ssh                                                                                                                                                      
```

Reply:
Fixed